React/CoreModules/RCTSourceCode.mm

```diff
@@ -7,7 +7,7 @@
 
 #import "RCTSourceCode.h"
 
-#import <FBReactNativeSpec/FBReactNativeSpec.h>
+#import <ReactCodegen/FBReactNativeSpec/FBReactNativeSpec.h>
```

Comment:
Commenting here but this applies to the whole PR.

Sadly, we can't merge this as it is, and we have to find a different approach to this problem. Specifically this is breaking our internal setup (i.e. building React Native iOS with Buck, which sadly we don't use is OSS).

Specifically here we fail internally with:

```
xplat/js/react-native-github/React/CoreModules/RCTSourceCode.mm:10:9: fatal error: 'ReactCodegen/FBReactNativeSpec/FBReactNativeSpec.h' file not found
#import <ReactCodegen/FBReactNativeSpec/FBReactNativeSpec.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
sure, i believed this change would break Buck builder 🤔 could you share more insight for how the Buck targets organized internally? from podspec of view, that's be little weird for `FBReactNativeSpec.podspec` doesn't have any source_files and `React-Codegen` puts source_files to `FBReactNativeSpec` import namespace. is it possible to merge these two?

Reply:
So ideally we can't change those `#import <FBReactNativeSpec/FBReactNativeSpec.h>` statements as our internal builds rely on them. I'd try to tweak the Podspecs as much as possible to tackle this.

Sadly this is a bigger problem, that is affecting how we do iOS in OSS, and we should look into this problem from a more holistic point of view.

For the sake of 0.68.x, if we could find a patch on the Podspec level, that would be awesome.

cc @cipolleschi @dmitryrykun for visiblity.

Reply:
So ideally we can't change those `#import <FBReactNativeSpec/FBReactNativeSpec.h>` statements as our internal builds rely on them. I'd try to tweak the Podspecs as much as possible to tackle this.

Sadly this is a bigger problem, that is affecting how we do iOS in OSS, and we should look into this problem from a more holistic point of view.

For the sake of 0.68.x, if we could find a patch on the Podspec level, that would be awesome.

cc @cipolleschi @dmitryrykun for visiblity.

Reply:
updated the pr to keep `#import <FBReactNativeSpec/FBReactNativeSpec.h> ` imports. please take a look when you get a chance.

this diff is for 0.68.0-rc.2 in case someone need to verify: https://gist.github.com/Kudo/17dde98d7721601697d2dc05ec1dcba1